browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -30,62 +39,339 @@ class RewardsDOMHandler : public WebUIMessageHandler,
 
  private:
   void HandleCreateWalletRequested(const base::ListValue* args);
-  void OnWalletCreated();
-  void OnWalletCreateFailed();
+  void GetWalletProperties(const base::ListValue* args);
+  void GetGrant(const base::ListValue* args);
+  void GetGrantCaptcha(const base::ListValue* args);
+  void GetWalletPassphrase(const base::ListValue* args);
+  void RecoverWallet(const base::ListValue* args);
+  void SolveGrantCaptcha(const base::ListValue* args);
+  void GetReconcileStamp(const base::ListValue* args);
+  void GetAddresses(const base::ListValue* args);
+  void SaveSetting(const base::ListValue* args);
+  void OnGetContentSiteList(std::unique_ptr<brave_rewards::ContentSiteList>, uint32_t record);
 
   // PaymentServiceObserver implementation
-  void OnWalletCreated(payments::PaymentsService* payment_service,
+  void OnWalletInitialized(brave_rewards::RewardsService* payment_service,
```

Comment:
Do we need `payment_service` to be `rewards_service`?

Reply:
done